packages/gatsby/cache-dir/loader.js

```diff
@@ -146,13 +151,25 @@ export class BaseLoader {
     // }
     this.pageDb = new Map()
     this.inFlightDb = new Map()
+
+    this.staticQueryDb = new Map()
+
+    this.inFlightPromisesByUrlDb = new Map()
```

Comment:
We can probably use the same `inFlightDb` for both (and `memoizedGet` then too).

Reply:
This is what I did after the refactor